src/item.cpp

```diff
@@ -5954,6 +5959,12 @@ bool item::process( player *carrier, const tripoint &pos, bool activate )
             ++it;
         }
     }
+
+    if ( has_flag("NANOFAB_TEMPLATE") && !has_var( "NANOFAB_ITEM_ID") ) {
+            itype_id nanofab_recipe = item_group::item_from( "nanofab_recipes" ).typeId();
+            set_var( "NANOFAB_ITEM_ID", nanofab_recipe );
+    }
+
```

Comment:
These lines assign a recipe to each individual nanofab template, but I'm not sure where to place them exactly. 

The current location works, but Im not convinced this is if the proper place for them. Any possible feedback on this?